src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -3421,6 +3421,165 @@ adjset_uniform(Node &res)
+
+        // points for half the triangular prisms
+        values_x[i * 20 + 8] = 1 + i * 2;       values_y[i * 20 + 8] = 1;   values_z[i * 20 + 8] = -1 + i * 2 + 2;
+        values_x[i * 20 + 9] = -1 + i * 2;      values_y[i * 20 + 9] = 1;   values_z[i * 20 + 9] = -1 + i * 2 + 2;
+        values_x[i * 20 + 10] = -1 + i * 2 + 2; values_y[i * 20 + 10] = 1;  values_z[i * 20 + 10] = 1 + i * 2 + 2;
+        values_x[i * 20 + 11] = 1 + i * 2;      values_y[i * 20 + 11] = -1; values_z[i * 20 + 11] = -1 + i * 2 + 2;
+        values_x[i * 20 + 12] = -1 + i * 2;     values_y[i * 20 + 12] = -1; values_z[i * 20 + 12] = -1 + i * 2 + 2;
+        values_x[i * 20 + 13] = -1 + i * 2 + 2; values_y[i * 20 + 13] = -1; values_z[i * 20 + 13] = 1 + i * 2 + 2;
+
+        // points for the other half
+        values_x[i * 20 + 14] = 1 + i * 2;      values_y[i * 20 + 14] = 1;  values_z[i * 20 + 14] = -1 + i * 2 + 2;
+        values_x[i * 20 + 15] = -1 + i * 2 + 2; values_y[i * 20 + 15] = 1;  values_z[i * 20 + 15] = -1 + i * 2;
+        values_x[i * 20 + 16] = -1 + i * 2 + 4; values_y[i * 20 + 16] = 1;  values_z[i * 20 + 16] = 1 + i * 2;
+        values_x[i * 20 + 17] = 1 + i * 2;      values_y[i * 20 + 17] = -1; values_z[i * 20 + 17] = -1 + i * 2 + 2;
+        values_x[i * 20 + 18] = -1 + i * 2 + 2; values_y[i * 20 + 18] = -1; values_z[i * 20 + 18] = -1 + i * 2;
+        values_x[i * 20 + 19] = -1 + i * 2 + 4; values_y[i * 20 + 19] = -1; values_z[i * 20 + 19] = 1 + i * 2;
+    }
+
+    chain_topo["type"] = "unstructured";
+    chain_topo["coordset"] = "coords";
+    chain_topo["elements/shape"] = "polyhedral";
+
+    // 6 cube faces + 5 prism faces * 2 prisms makes 16
+    chain_topo["elements/connectivity"].set(conduit::DataType::int64(howmany * 16));
+    int64* connec = chain_topo["elements/connectivity"].value();
+    for (int i = 0; i < howmany * 16; i ++)
+    {
+        // our faces are specified in order and no faces are reused
+        connec[i] = i;
+    }
+
+    // this is 3 because every time howmany is increased by 1, 3 more polyhedra are added,
+    // the cube and two prisms
+    chain_topo["elements/sizes"].set(conduit::DataType::int64(howmany * 3));
+    int64* sizes = chain_topo["elements/sizes"].value();
+    for (int i = 0; i < howmany * 3; i ++)
+    {
+        // this ensures that sizes will be of the form {6,5,5, 6,5,5, 6,5,5, ..., 6,5,5}
+        sizes[i] = ((i % 3) > 0) ? 5 : 6;
+    }
+
+    chain_topo["elements/offsets"].set(conduit::DataType::int64(howmany * 3));
+    int64* offsets = chain_topo["elements/offsets"].value();
+    offsets[0] = 0;
+    for (int i = 1; i < howmany * 3; i ++)
+    {
+        offsets[i] = offsets[i - 1] + sizes[i - 1];
+    }
+
+    chain_topo["subelements/shape"] = "polygonal";
+    // 24 points -> 6 sides -> a cube 
+    // 18 points -> 5 sides -> triangular prism 
+    // so 24 + 18 * 2 = 60
+    chain_topo["subelements/connectivity"].set(conduit::DataType::int64(howmany * 60));
```

Comment:
I'd like to have a similar recommendation for the 60 here, but it's a bit hard to unpack as it involves a cascade of values. Maybe something high level will suffice, like:

```
const index_t num_points_per_hex = 24;
const index_t num_points_per_triprism = 18;
const index_t num_points_per_chainpair = num_points_per_hex + 2 * num_points_per_triprism;
```